src/main/java/org/spongepowered/api/entity/Item.java

```diff
@@ -25,17 +25,88 @@
 
 package org.spongepowered.api.entity;
 
+import com.google.common.base.Optional;
+import org.spongepowered.api.entity.player.Player;
 import org.spongepowered.api.item.inventory.ItemStack;
 
+import javax.annotation.Nullable;
+
 /**
  * Represents an Item entity.
  */
 public interface Item extends Entity {
 
     /**
-     * Get the {@link ItemStack} that this entity represents.
+     * Get the {@link ItemStack} that this Item represents.
```

Comment:
I would say you should either use `{@link Item}` or `item`, but not just the class name.